cmd/new.go

```diff
@@ -108,6 +109,7 @@ func init() {
 	rootCmd.AddCommand(newCmd)
 
 	newCmd.Flags().DurationVarP(&expires, "expires", "x", defaultExpiry, "Secret will be deleted from the server after specified duration, supported units: s,m,h")
+	newCmd.Flags().StringVar(&region, "region", defaultRegion, "The region where secret should be created, supported regions: us-east-1,eu-central-1")
```

Comment:
I think region names should be decoupled from the exact AWS region name because from a DX standpoint I just want to know it's in `US East`, for example. WDYT?

Reply:
Agreed, but also need to find a good compromise RE user input. Not sure we'd want people to write `--region "EU West"`. I'm happy to simplify to `eu` and `us` for now, not sure anyone cares if it's east or west - we simply use the most sensible defaults.

Reply:
Settled on eu and us for now.